autogpt_platform/frontend/src/components/agptui/NavbarLink.tsx

```diff
@@ -12,17 +12,18 @@ import { usePathname } from "next/navigation";
 interface NavbarLinkProps {
   name: string;
   href: string;
+  className?: string;
 }
 
-export const NavbarLink = ({ name, href }: NavbarLinkProps) => {
+export const NavbarLink = ({ name, href, className }: NavbarLinkProps) => {
   const pathname = usePathname();
   const parts = pathname.split("/");
   const activeLink = "/" + (parts.length > 2 ? parts[2] : parts[1]);
 
   return (
     <Link href={href} data-testid={`navbar-link-${name.toLowerCase()}`}>
       <div
-        className={`h-[48px] px-5 py-4 ${
+        className={`h-[48px] px-5 py-4 ${className} ${
```

Comment:
1. Use `cn` to make sure `className` takes precedence over the hard-coded classes
2. Shouldn't this be on `Link` two lines up?

Reply:
This shouldn’t be here at all imo but I’m just trying to apply it where it looks like it was supposed to go